.pre-commit-config.yaml

```diff
@@ -3,3 +3,4 @@ repos:
     rev: 22.3.0
     hooks:
       - id: black
+        exclude: ./migrations/
```

Comment:
On the Berlin repo we have an automated action on push (https://github.com/SuperCoopBerlin/tapir/blob/master/.github/workflows/push.yml) that checks for black formatting on all files including the migrations, I would leave them formatted.

Reply:
Yes I will change it back.